README.md

```diff
@@ -8,15 +8,16 @@ GitHub Actions are pretty damn cool, but lord knows the official runners are slo
 
 ## Table of contents
 
-- [Awesome GitHub Actions runners 🏃🏾‍♀️🤖](#awesome-github-actions-runners-️)
+- [Awesome GitHub Actions runners ⚡🤖](#awesome-github-actions-runners-)
 	- [Table of contents](#table-of-contents)
 	- [List of providers](#list-of-providers)
 		- [Namespace ⭐](#namespace-)
 		- [BuildJet ⭐](#buildjet-)
 		- [Actuated ⭐](#actuated-)
 		- [WarpBuild](#warpbuild)
-		- [Cirun](#cirun)
 		- [Ubicloud](#ubicloud)
+		- [RunsOn](#runson)
+		- [Cirun](#cirun)
```

Comment:
I am curious, why this ordering is changed? Also what's the purpose of ⭐ in front of the first 3?